deltachat-ios/Controller/QrViewController.swift

```diff
@@ -140,17 +140,28 @@ class QrViewController: UITableViewController, QrCodeReaderDelegate {
         switch state {
         case DC_QR_ASK_VERIFYCONTACT:
             let nameAndAddress = DcContact(id: qrParsed.id).nameNAddr
-            joinSecureJoin(alertMessage: String.localizedStringWithFormat(String.localized("qrscan_ask_fingerprint_ask_oob"), nameAndAddress), code: code)
+            joinSecureJoin(alertMessage: String.localizedStringWithFormat(String.localized("ask_start_chat_with"), nameAndAddress), code: code)
+
         case DC_QR_ASK_VERIFYGROUP:
-            if let group = qrParsed.text1?.replacingOccurrences(of: "+", with: " ") {
-                joinSecureJoin(alertMessage: String.localizedStringWithFormat(String.localized("qrscan_ask_join_group"), group), code: code)
-            }
+            let groupName = qrParsed.text1 ?? "ErrGroupName"
```

Comment:
ErrGroupName will displayed to the user if text1 is empty? Why not something localized like "unknown group"?

Reply:
because it is no error the user should be aware of at all. when we get the info "group" but no name, this should be fixed in the core, it normally cannot happen, no need to give work to the translators for that.

Reply:
ok